ui/src/shared/components/time_zones/TimeZoneToggle.tsx

```diff
@@ -0,0 +1,43 @@
+// Libraries
+import React, {FunctionComponent} from 'react'
+
+// Actions
+import * as appActions from 'src/shared/actions/app'
+
+// Components
+import {Radio} from 'src/reusable_ui'
+
+// Types
+import {TimeZones} from 'src/types'
+
+interface StateProps {
+    timeZone: TimeZones
+    onSetTimeZone: typeof appActions.setTimeZone
+}
+
+const TimeZoneToggle: FunctionComponent<StateProps> = ({onSetTimeZone, timeZone}) => {
+    return (
+      <Radio>
+        <Radio.Button
+          id="influxql-source"
+          titleText="UTC"
+          value={TimeZones.UTC}
+          onClick={onSetTimeZone}
+          active={timeZone === TimeZones.UTC}
+        >
+            UTC
+        </Radio.Button>
+        <Radio.Button
+          id="flux-source"
+          titleText="Local"
```

Comment:
maybe these title texts could be more descriptive like "set time zone display to local/utc"